client/components/forms/multi-checkbox/index.jsx

```diff
@@ -43,24 +43,34 @@ var MultiCheckbox = module.exports = React.createClass({
 		} );
 
 		event.stopPropagation();
-	},
+	};
 
-	getCheckboxElements: function() {
-		var checked = this.props.checked || this.state.initialChecked;
+	getCheckboxElements() {
+		const checked = this.props.checked || this.state.initialChecked;
 
-		return this.props.options.map( function( option ) {
-			var isChecked = checked.indexOf( option.value ) !== -1;
+		return this.props.options.map( ( option ) => {
+			const isChecked = checked.indexOf( option.value ) !== -1;
 
 			return (
 				<label key={ option.value }>
-					<input name={ this.props.name + '[]' } type="checkbox" value={ option.value } checked={ isChecked } onChange={ this.handleChange } disabled={ this.props.disabled } />
+					<input
+						name={ this.props.name + '[]' }
+						type="checkbox" value={ option.value }
```

Comment:
`value` should be on its own line. Thinking this might be a good guideline to include in #9037.